infrastructure/lambda/template.yaml

```diff
@@ -817,97 +536,16 @@ Resources:
 #                                                                  #
 ####################################################################
 
-  DCSResponseTable:
-    Type: AWS::DynamoDB::Table
-    # checkov:skip=CKV_AWS_28: Point in time recovery is not necessary for this table.
-    Properties:
-      TableName: !Sub "dcs-response-${AWS::StackName}"
-      BillingMode: "PAY_PER_REQUEST"
-      AttributeDefinitions:
-        - AttributeName: "resourceId"
-          AttributeType: "S"
-      KeySchema:
-        - AttributeName: "resourceId"
-          KeyType: "HASH"
-      TimeToLiveSpecification:
-        AttributeName: "ttl"
-        Enabled: true
-      SSESpecification:
-        # checkov:skip=CKV_AWS_119: Implement Customer Managed Keys in PYIC-1391
-        SSEEnabled: true
-        SSEType: KMS
-
-  AuthCodesTable:
-    Type: AWS::DynamoDB::Table
-    # checkov:skip=CKV_AWS_28: Point in time recovery is not necessary for this table.
-    Properties:
-      TableName: !Sub "auth-codes-${AWS::StackName}"
-      BillingMode: "PAY_PER_REQUEST"
-      AttributeDefinitions:
-        - AttributeName: "authCode"
-          AttributeType: "S"
-      KeySchema:
-        - AttributeName: "authCode"
-          KeyType: "HASH"
-      TimeToLiveSpecification:
-        AttributeName: "ttl"
-        Enabled: true
-      SSESpecification:
-        # checkov:skip=CKV_AWS_119: Implement Customer Managed Keys in PYIC-1391
-        SSEEnabled: true
-        SSEType: KMS
-
-  AccessTokenTable:
-    Type: AWS::DynamoDB::Table
-    # checkov:skip=CKV_AWS_28: Point in time recovery is not necessary for this table.
+  DocumentCheckResultTable:
```

Comment:
Can we also make sure we have noted somewhere DB table structure and names that have changed and what there new counter parts are. We will still likely have to do this final migration late at night to avoid disruption